README.md

```diff
@@ -56,7 +56,15 @@ node_api_ip_port_address: your-public-ip:9010
 enable_node_api: true
 ```
 
-Then onboard your operator into our AVS
+Configure 2 env var for your ECDSA and BLS password. Recall that these are
+generated when you onboard your operator to EigenLayer.
+
+```
+export OPERATOR_BLS_KEY_PASSWORD=
+export OPERATOR_ECDSA_KEY_PASSWORD=
+```
+
+Now, we can start the registration process.
```

Comment:
Let’s replace the old `OAK` with `Ava Protocol` in this doc, for example, "OAK AVS" -> "Ava Protocol AVS"

The `oak-avs` name needs to change too, but we can do another sweep in a separate PR.